commcare_connect/opportunity/api/views.py

```diff
@@ -24,10 +25,27 @@
 )
 from commcare_connect.users.helpers import create_hq_user_and_link
 
+logger = logging.getLogger(__name__)
+
+
+class OpportunityViewSetPermission(IsAuthenticated):
+    def has_permission(self, request, view):
+        user_has_permission = bool(request.user and request.user.is_authenticated)
```

Comment:
nit: it would be a little more robust to call the super function here, on the off chance this changes in future versions